README.md

```diff
@@ -1,6 +1,18 @@
 # TrackLooper
 
-## Quick start guide
+## Quick Start
+
+    git clone git@github.com:SegmentLinking/TrackLooper.git
+    cd TrackLooper/
+    source setup.sh
+    # source setup_cgpu.sh # if you are on cgpu-1
+    sdl_make_tracklooper -mc8
+    sdl -i PU200 -o LSTNtuple.root
+    createPerfNumDenHists -i LSTNtuple.root -o LSTNumDen.root
+    lst_plot_performance.py LSTNumDen.root -t "myTag"
```

Comment:
I wanted to separate them because in case we wish to create a few set of plots, i don't want to re-run the whole thing.
The code running to produce the entire set of histograms for all possibilities takes 9 seconds.
The code running the plotter takes 1-2 minutes for a standard set of plots
The code if you specify to generate just a few set of plots, can run in <few seconds.